packages/@lwc/style-compiler/src/serialize.ts

```diff
@@ -56,10 +57,21 @@ export default function serialize(result: Result, config: Config): string {
     const serializedStyle = serializeCss(result, collectVarFunctions).trim();
 
     if (serializedStyle) {
-        // inline function
-        buffer += `function stylesheet(${HOST_SELECTOR_IDENTIFIER}, ${SHADOW_SELECTOR_IDENTIFIER}, ${SHADOW_DOM_ENABLED_IDENTIFIER}) {\n`;
-        buffer += `  return ${serializedStyle};\n`;
-        buffer += `}\n`;
+        // In the case where adopted stylesheets (aka construtable stylesheets) are supported,
```

Comment:
This comment is out of date now. The fact that `createCachingCssGenerator` creates a constructible stylesheet is an internal detail.